--- datanode.conf.example.orig	2023-04-12 16:36:18.208456594 +0200
+++ datanode.conf.example	2023-04-12 16:40:36.897279746 +0200
@@ -1,15 +1,15 @@
 # defaults
 # locations of bin, data and log directories
-opensearch_location = dist/opensearch-2.4.1
-opensearch_config_location = config
-opensearch_data_location = data
-opensearch_logs_location = logs
+opensearch_location = /usr/share/graylog-datanode/dist/opensearch-2.4.1
+opensearch_config_location = /var/lib/graylog-datanode/opensearch/config
+opensearch_data_location = /var/lib/graylog-datanode/opensearch/data
+opensearch_logs_location = /var/log/graylog-datanode/opensearch
 
 # connection to the mongodb, shared with the Graylog server
 mongodb_uri = mongodb://localhost/graylog
 
 # location of the node_id file
-node_id_file = data/node-id
+node_id_file = /etc/graylog/datanode/node-id
 
 # You MUST set a secret to secure/pepper the stored user passwords here. Use at least 64 characters.
 # Generate one by using for example: pwgen -N 1 -s 96
