--- datanode.conf.example.orig	2023-09-25 10:26:00.000000000 +0200
+++ datanode.conf.example	2023-09-25 14:07:05.585381994 +0200
@@ -1,21 +1,21 @@
-# defaults
+# Graylog Data Node configuration
 
 # Root directory of opensearch distribution
-opensearch_location = dist/
+opensearch_location = /usr/share/graylog-datanode/dist
 
 # data-node config location
 config_location = /etc/graylog/datanode
 
 # locations of opensearch bin, data and log directories 
-opensearch_config_location = config
-opensearch_data_location = data
-opensearch_logs_location = logs
+opensearch_config_location = /var/lib/graylog-datanode/opensearch/config
+opensearch_data_location = /var/lib/graylog-datanode/opensearch/data
+opensearch_logs_location = /var/log/graylog-datanode/opensearch
 
 # connection to the mongodb, shared with the Graylog server
 mongodb_uri = mongodb://localhost/graylog
 
 # location of the node_id file
-node_id_file = data/node-id
+node_id_file = /etc/graylog/datanode/node-id
 
 # You MUST set a secret to secure/pepper the stored user passwords here. Use at least 64 characters.
 # Generate one by using for example: pwgen -N 1 -s 96
@@ -23,13 +23,19 @@
 # Changing this value after installation will render all user sessions and encrypted values in the database invalid. (e.g. encrypted access tokens)
 password_secret =
 
+# The default root user is named 'admin'
+#root_username = admin
+
+# You MUST specify a hash password for the root user (which you only need to initially set up the
+# system and in case you lose connectivity to your authentication backend)
+# This password cannot be changed using the API or via the web interface. If you need to change it,
+# modify it in this file.
+# Create one by using for example: echo -n yourpassword | shasum -a 256
+# and put the resulting hash value into the following line
+root_password_sha2 =
 
 # Certificate bundles for transport and http layer security. You can generate these using built-in certutil tool
-transport_certificate = datanode-transport-certificates.p12
-transport_certificate_password = password
-http_certificate = datanode-http-certificates.p12
-http_certificate_password = password
-
-# initial credentials for the admin user in the REST API of Opensearch
-rest_api_username = admin
-rest_api_password = admin
+#transport_certificate = datanode-transport-certificates.p12
+#transport_certificate_password = password
+#http_certificate = datanode-http-certificates.p12
+#http_certificate_password = password
