--- graylog-radio.conf.example.orig	2015-02-13 17:33:01.922658691 +0100
+++ graylog-radio.conf.example	2015-02-13 17:33:44.746570547 +0100
@@ -6,7 +6,7 @@
 transport_type = amqp
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-radio/plugin
 
 # REST URI of a Graylog server node this radio will register at.
 graylog2_server_uri = http://your-graylog2-server.example.org:12900/
