--- graylog2-radio.conf.example.orig	2014-05-23 15:40:19.000000000 +0000
+++ graylog2-radio.conf.example	2014-06-12 10:09:12.965311343 +0000
@@ -1,6 +1,6 @@
 # The auto-generated node ID will be stored in this file and read after restarts. It is a good idea
 # to use an absolute file path here if you are starting graylog2-radio from init scripts or similar.
-node_id_file = /etc/graylog2-radio-node-id
+node_id_file = /etc/graylog2/radio/node-id
 
 # Use AMQP ("amqp") or Kafka ("kafka") as message transport?
 transport_type = amqp
