--- graylog.conf.example.orig	2016-12-21 13:55:37.000000000 +0100
+++ graylog.conf.example	2017-01-05 13:34:47.619337710 +0100
@@ -74,7 +74,7 @@
 #root_timezone = UTC
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 # REST API listen URI. Must be reachable by other Graylog server nodes if you run a cluster.
 # When using Graylog Collectors, this URI will be used to receive heartbeat messages and must be accessible for all collectors.
@@ -391,7 +391,7 @@
 #   If you create a seperate partition for the journal files and use a file system creating directories like 'lost+found'
 #   in the root directory, you need to create a sub directory for your journal.
 #   Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
-message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
@@ -532,7 +532,7 @@
 #content_packs_loader_enabled = true
 
 # The directory which contains content packs which should be loaded on the first start of Graylog.
-#content_packs_dir = data/contentpacks
+content_packs_dir = /usr/share/graylog-server/contentpacks
 
 # A comma-separated list of content packs (files in "content_packs_dir") which should be applied on
 # the first start of Graylog.
