--- graylog.conf.example.orig	2015-02-13 14:53:40.000000000 +0100
+++ graylog.conf.example	2015-02-13 17:30:38.838954145 +0100
@@ -30,7 +30,7 @@
 #root_timezone = UTC
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 # REST API listen URI. Must be reachable by other graylog2-server nodes if you run a cluster.
 rest_listen_uri = http://127.0.0.1:12900/
@@ -245,7 +245,7 @@
 
 # The directory which will be used to store the message journal. The directory must me exclusively used by Graylog and
 # must not contain any other files than the ones created by Graylog itself.
-message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
