--- graylog.conf.example.orig	2018-09-14 18:37:48.000000000 +0200
+++ graylog.conf.example	2018-09-14 19:32:35.014475170 +0200
@@ -76,15 +76,15 @@
 # Set the bin directory here (relative or absolute)
 # This directory contains binaries that are used by the Graylog server.
 # Default: bin
-bin_dir = bin
+bin_dir = /usr/share/graylog-server/bin
 
 # Set the data directory here (relative or absolute)
 # This directory is used to store Graylog server state.
 # Default: data
-data_dir = data
+data_dir = /var/lib/graylog-server
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 ###############
 # HTTP settings
@@ -468,7 +468,7 @@
 #   If you create a seperate partition for the journal files and use a file system creating directories like 'lost+found'
 #   in the root directory, you need to create a sub directory for your journal.
 #   Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
-message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
