--- graylog.conf.example.orig	2016-03-10 17:11:21.570987281 +0100
+++ graylog.conf.example	2016-03-10 17:12:09.574908506 +0100
@@ -31,7 +31,7 @@
 #root_timezone = UTC
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 # REST API listen URI. Must be reachable by other Graylog server nodes if you run a cluster.
 # When using Graylog Collectors, this URI will be used to receive heartbeat messages and must be accessible for all collectors.
@@ -298,7 +298,7 @@
 
 # The directory which will be used to store the message journal. The directory must me exclusively used by Graylog and
 # must not contain any other files than the ones created by Graylog itself.
-message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
@@ -434,7 +434,7 @@
 #content_packs_loader_enabled = true
 
 # The directory which contains content packs which should be loaded on the first start of Graylog.
-#content_packs_dir = data/contentpacks
+content_packs_dir = /usr/share/graylog-server/contentpacks
 
 # A comma-separated list of content packs (files in "content_packs_dir") which should be applied on
 # the first start of Graylog.
