--- graylog.conf.example.orig	2024-03-01 10:21:34.773290089 +0100
+++ graylog.conf.example	2024-03-01 10:23:43.200807725 +0100
@@ -78,14 +78,14 @@
 # Set the bin directory here (relative or absolute)
 # This directory contains binaries that are used by the Graylog server.
 # Default: bin
-bin_dir = bin
+bin_dir = /usr/share/graylog-server/bin
 
 # Set the data directory here (relative or absolute)
 # This directory is used to store Graylog server state.
-data_dir = data
+data_dir = /var/lib/graylog-server
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 ###############
 # HTTP settings
@@ -524,7 +524,7 @@
 #   in the root directory, you need to create a sub directory for your journal.
 #   Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
 # Default: <data_dir>/journal
-#message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
