--- graylog.conf.example.orig	2018-09-14 18:37:48.000000000 +0200
+++ graylog.conf.example	2018-09-14 19:32:35.014475170 +0200
@@ -76,15 +76,15 @@
 # Set the bin directory here (relative or absolute)
 # This directory contains binaries that are used by the Graylog server.
 # Default: bin
-bin_dir = bin
+bin_dir = /usr/share/graylog-server/bin
 
 # Set the data directory here (relative or absolute)
 # This directory is used to store Graylog server state.
 # Default: data
-data_dir = data
+data_dir = /var/lib/graylog-server
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog-server/plugin
 
 ###############
 # HTTP settings
@@ -468,7 +468,7 @@
 #   If you create a seperate partition for the journal files and use a file system creating directories like 'lost+found'
 #   in the root directory, you need to create a sub directory for your journal.
 #   Otherwise Graylog will log an error message that the journal is corrupt and Graylog will not start.
-message_journal_dir = data/journal
+message_journal_dir = /var/lib/graylog-server/journal
 
 # Journal hold messages before they could be written to Elasticsearch.
 # For a maximum of 12 hours or 5 GB whichever happens first.
@@ -632,17 +632,6 @@
 # The default cache time for dashboard widgets. (Default: 10 seconds, minimum: 1 second)
 #dashboard_widget_default_cache_time = 10s
 
-# Automatically load content packs in "content_packs_dir" on the first start of Graylog.
-#content_packs_loader_enabled = true
-
-# The directory which contains content packs which should be loaded on the first start of Graylog.
-#content_packs_dir = data/contentpacks
-
-# A comma-separated list of content packs (files in "content_packs_dir") which should be applied on
-# the first start of Graylog.
-# Default: empty
-content_packs_auto_load = grok-patterns.json
-
 # For some cluster-related REST requests, the node must query all other nodes in the cluster. This is the maximum number
 # of threads available for this. Increase it, if '/cluster/*' requests take long to complete.
 # Should be http_thread_pool_size * average_cluster_size if you have a high number of concurrent users.
