--- graylog2.conf.example.orig	2014-05-08 13:42:55.416580849 +0200
+++ graylog2.conf.example	2014-05-08 13:43:52.804470954 +0200
@@ -4,7 +4,7 @@
 
 # The auto-generated node ID will be stored in this file and read after restarts. It is a good idea
 # to use an absolute file path here if you are starting graylog2-server from init scripts or similar.
-node_id_file = /etc/graylog2-server-node-id
+node_id_file = /var/lib/graylog2/graylog2-server-node-id
 
 # You MUST set a secret to secure/pepper the stored user passwords here. Use at least 64 characters.
 # Generate one by using for example: pwgen -s 96
@@ -20,7 +20,7 @@
 root_password_sha2 =
 
 # Set plugin directory here (relative or absolute)
-plugin_dir = plugin
+plugin_dir = /usr/share/graylog2-server/plugin
 
 # REST API listen URI. Must be reachable by other graylog2-server nodes if you run a cluster.
 rest_listen_uri = http://127.0.0.1:12900/
