--- config/collector.conf.example.orig	2015-07-20 15:41:04.858068011 +0200
+++ config/collector.conf.example	2015-07-20 15:42:16.849800755 +0200
@@ -13,7 +13,7 @@
 // an id will be generated and written to that file. If it exists, it is expected
 // to contain a single string without spaces which will be used for the id.
 // Defaults to "file:config/collector-id" if not specified.
-collector-id = "file:config/collector-id"
+collector-id = "file:/etc/graylog/collector/collector-id"
 
 //inputs {
 //  // A simple file input that follows /var/log/syslog.
